fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/services/VersionServiceImplTest.java

```diff
@@ -169,15 +171,54 @@ public void testRemoveUnknownVersion() throws RepositoryException {
 
     @Test
     public void testMixinCreationWhenExplicitlyVersioning() throws RepositoryException {
+        testCreateWithValidLabel("LABEL");
+    }
+
+    @Test
+    public void testCreateWithLabelv001() throws RepositoryException {
+        testCreateWithValidLabel("v0.0.1");
+    }
+
+    @Test
+    public void testCreateWithLabelWithSpace() throws RepositoryException {
+        testCreateWithValidLabel("version v1234");
+    }
+
+
+    private void testCreateWithValidLabel(final String label) throws RepositoryException {
```

Comment:
One way here is to wrap the lines where an exception might be thrown with `try {...} catch (Exception e) {fail(e.getMessage());}`.